### PR TITLE
add more notifications

### PR DESCRIPTION
**Pull request by ngokevin**

### CODE REVIEW

File: src/media/js/addon/actions/comm.js [hunk 1]
```diff
@@ -68,6 +70,8 @@ export function submitNote(threadId, versionId, body, noteType) {
           note: res.body,
           versionId
         }));
+        dispatch(notificationActions.queue(
+          'Your note has been successfully submitted.'));
```
Comment (chuckharmston) [inline src/media/js/addon/actions/comm.js]: Do we use "note" as a user-facing term? This reads a little strange.

**Comment by chuckharmston** [comment]
r+

File: src/media/js/addon/actions/comm.js (hunk 1 above)
Comment (ngokevin) [inline src/media/js/addon/actions/comm.js]: i'll say message

**Pull request merged**